### PR TITLE
[LUMI-33] Move env vars to settings

### PR DESCRIPTION
**Pull request by javiermtorres**
This PR moves the environment variables retrieval to `settings.py`.

### CODE REVIEW

File: lumigator/python/mzai/backend/settings.py [hunk 1]
```diff
@@ -54,6 +56,32 @@ class BackendSettings(BaseSettings):
...
     SUMMARIZER_WORK_DIR: str | None = None
 
+    def inherit_ray_env(self, runtime_env_vars: Mapping[str, str]):
+        for env_var_name in self.RAY_WORKER_ENV_VARS:
+            env_var = os.environ.get(env_var_name, None)
+            if env_var is not None:
+                runtime_env_vars[env_var_name] = env_var
+
+    @computed_field
+    @property
```
Comment (chainlink) [inline lumigator/python/mzai/backend/settings.py:66]: I think we can just use a regular variable here?

File: lumigator/python/mzai/backend/settings.py [hunk 2]
```diff
@@ -54,6 +56,32 @@ class BackendSettings(BaseSettings):
...
+    def OAI_API_KEY(self) -> str:  # noqa: N802
+        return os.environ["OPENAI_API_KEY"]
+
+    @computed_field
+    @property
+    def MISTRAL_API_KEY(self) -> str:  # noqa: N802
```
[the remark this answers is not in the document]
Comment (javiermtorres) [reply lumigator/python/mzai/backend/settings.py:72]: Yes if the env vars will not be reloaded dynamically (I assumed we'd like hot reload at some point). @veekaybee @aittalam thoughts on this????

[the remark this answers is not in the document]
Comment (chainlink) [reply lumigator/python/mzai/backend/settings.py:72]: Hot reloading is an interesting idea! We don't do that currently and most of the other env vars don't currently support it. Perhaps add a ticket to the backlog to explore it?

[the remark this answers is not in the document]
Comment (javiermtorres) [reply lumigator/python/mzai/backend/settings.py:72]: Ok, so:
* Add hot reloading as potential feature
* Replace with regular variables

**Review by chainlink — APPROVED** [review]

**Pull request merged**